### PR TITLE
Add Joel PPP

### LINKED ISSUE #221: Claim all code: yAOwzers
Use the `@@author` tag to claim all code appropriately.

### PR DESCRIPTION
**Pull request by yAOwzers**

### CODE REVIEW

**Review by Lusi711 — COMMENTED** [review]
You might want to elaborate on this further e.g. Justification, Highlights
https://nus-cs2113-ay2021s1.github.io/addressbook-level3/team/johndoe.html

File: docs/team/joel.md [hunk 1]
```diff
@@ -2,5 +2,17 @@
...
+- New Feature: Implemented the ability to mark existing tasks in the tasklist as done.
+- New Feature: Implemented a personalised message generator that encourages the user after certain successful input.
+- Code Contributed: [Reposense link](https://nus-cs2113-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=yAOwzers&sort=groupTitle&sortWithin=title&since=2020-09-27&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other)
+- Documentation: 
+    - User Guide: 
+        - Included details for `mark as done` feature. [#84](https://github.com/AY2021S1-CS2113T-T12-3/tp/pull/84)
+    - Developer Guide:
+        - Added implementation details of the `mark as done` feature. [#145](https://github.com/AY2021S1-CS2113T-T12-3/tp/pull/145)
+        - Added manual test cases under `appendix F`. [#129](https://github.com/AY2021S1-CS2113T-T12-3/tp/pull/129)
+
```
Comment (Lusi711) [inline docs/team/joel.md]: Include some of the PRs you reviewed and bugs you reported of a grp

Comment (yAOwzers) [reply docs/team/joel.md]: Oh why thank you will do

File: docs/team/joel.md [hunk 2]
```diff
@@ -2,5 +2,44 @@
...
 
 ### Summary of Contributions
+
+- **New Feature:** Implemented the ability to mark existing tasks in the tasklist as done.
+    - What it does: Enables the user to mark their listed tasks in the tasklist as done. 
+    - Justification: The user's productivity is improved when the user is provided with visuals that the task that they had listed in the tasklist is finished.
+    
+- **New Feature:** Implemented a personalised message generator that encourages the user after certain successful input.
+    - What it does: A personalised message will be printed after certain user inputs. 
+    - Justification: This features serves as a form of encouragement to the user. Users who use Zer0Note will be with the intention of keeping track of their deadlines and notes. This might be a very dull task, and having a personalised message sent to the user to encourage them aims to lighten up the mood of the user.
```
Comment (franceneee) [inline docs/team/joel.md]: nice!

**Review by franceneee — APPROVED** [review]

**Comment by franceneee** [comment]
omg, I just realised but @yAOwzers , your ppp should be called yAOwzers.md I think...

**Comment by yAOwzers** [comment]
FATS

**Review by longngng — APPROVED** [review]

**Pull request merged**